contracts/strategy/liquity/LiquityStrategy.sol

```diff
@@ -147,15 +156,33 @@ contract LiquityStrategy is
     }
 
     /// @inheritdoc IStrategy
-    /// @notice ETH & LQTY rewards of the strategy waiting to be claimed in the liquity stability pool are not included
+    /// @notice LQTY rewards of the strategy waiting to be claimed in the liquity stability pool are not included
+    /// @notice but the ETH rewards are included
     function investedAssets()
         public
         view
         virtual
         override(IStrategy)
         returns (uint256)
     {
-        return stabilityPool.getCompoundedLUSDDeposit(address(this));
+        uint256 ethBalance = address(this).balance;
+        uint256 ethBalanceInLusd;
+        // need to do this because the get_exchange_amount method reverts if ethBalance is zero
+        if (ethBalance != 0) {
```

Comment:
I would suggest changing this condition to `ethBalance == 0` and returning `stabilityPool.getCompoundedLUSDDeposit(address(this))`. It would also make Slither happier since you would not have to declare a variable which is not used in the current else case

Reply:
Fixed